InvenTree/part/filters.py

```diff
@@ -71,6 +71,22 @@ def annotate_total_stock(reference: str = ''):
     )
 
 
+def annotate_last_updated(reference: str = ''):
+    """Annotate the last updated date for each supplier part in a queryset
+
+    - Include supplier part's updated field
+    - Include supplier part's availability_updated field
+    - Include supplier part's price breaks
+    """
+
+    return models.Max(
+        F('updated'),
+        F('availability_updated'),
+        # TODO Add price breaks
```

Comment:
Question: What do you want to order by on price breaks?

Reply:
Because a price break may be the most recent update to the supplier part.